### ISSUE TITLE
I see no govee objects after adapter installation

### ISSUE THREAD

**Issue by MAD724** [boergegrunicke/ioBroker.govee-local#82]
Hello, I am quite new with iobroker and currently struggeling with the goove-local adapter. I have installed adapter version 0.2.5 and I have as well switched on the LAN-Steuerung inside the govee App for my device H619A. The control via the app works well but I do not know how to add the device to iobroker. I think I need to add somewhere in the adapter my govee password. But where? In the adapter instance there is more or less nothing to add or midify. Under objects, I have govee-local with the 0-info-connectoin tree but no device or content. I hope it is easy and I am just blind. Thanks in advance

**Comment by MAD724** [comment]
Additional info: iobroker is running on Pi3B. I have enabled extended logging as you have adviced in another threat and selected current IP and not all IP. But the adapter does not search or at least does not find the device.
node: v18.19.0, js-controller: 5.0.19

**Comment by MAD724** [comment]
small correction but it should not make a difference: iobroker is running on Pi4

**Comment by boergegrunicke** [comment]
@MAD724 There is no password needed. The local communication is secured by the fact, that it is using local Multicasts, that will not leave the network.
These Multicasts could be the tricky part about it. Do you have the pi and your govee lights in the same network (same broadcast domain)?
You mentioned that you enabled the extended logging, is there anything from the govee-local adapter inside the log or just empty?

**Comment by MAD724** [comment]
I have solved the problem. It was due to the wifi connection of the app. When I have installed the govee app on my mobile, the mobile has been in 5 Ghz connection. Then I have seen that this is not supported and switched to 2.4 Ghz. But this has not worked properly obviously. I have deinstalled everything, then switched to 2.4 Ghz and then installed the app again. Now it works. Thanks for your efforts.
